File: simple-web-server/src/main/java/de/thi/informatik/gdp2/simplewebapp/model/Message.java
package de.thi.informatik.gdp2.simplewebapp.model;

import java.text.SimpleDateFormat;
import java.util.Date;

public class Message {
	private long time;
	private String user;
	private String message;
	
	public Message(String user, String message) {
		this.user = user;
		this.message = message;
		this.time = System.currentTimeMillis();
	}
	
	public String getMessage() {
		return message;
	}
	
	public String getUser() {
		return user;
	}
	
	public long getTime() {
		return time;
	}

    public String toFormattedTime() {
        SimpleDateFormat sdf = new SimpleDateFormat("hh:mm:ss");
        return sdf.format(new Date(time));
    }
	
	@Override
	public String toString() {
		return this.getUser() + "(" + this.toFormattedTime() + "): " + this.getMessage();
	}
}

File: simple-web-server-kotlin/src/main/kotlin/de/thi/informatik/gdp2/simplewebserverkotlin/model/Message.kt
package de.thi.informatik.gdp2.simplewebserverkotlin.model

import java.text.SimpleDateFormat
import java.util.Date

class Message {
    val time: Long
        get() = field
    val user: String
        get() = field
    val message: String
        get() = field

    constructor(user: String, message: String) {
        this.time = System.currentTimeMillis()
        this.user = user
        this.message = message
    }

    private fun toFormattedTime(): String =
        SimpleDateFormat("hh:mm:ss").format(Date(time))

    override fun toString(): String =
        "${user} (${toFormattedTime()}: ${message}"
}